src/zope/i18n/locales/xmlfactory.py

```diff
@@ -771,7 +771,7 @@ def _extractCalendars(self, dates_node):
 
           >>> factory = LocaleFactory(None)
           >>> from xml.dom.minidom import parseString
-          >>> xml = _u('''
+          >>> xml = (u'''
```

Comment:
You could drop the parentheses in these cases too.  (Everywhere, not just here.)

Reply:
Keeping them actually made the diff smaller :) (and was an easier mechanical transform for multi-line strings).

Reply:
FWIT, there are also places like https://github.com/zopefoundation/zope.i18n/pull/25/files#diff-7c85cf7577ef949cc20b86e1a2260db3R263 where we have multi-line string literals composed by allowing adjacent one-line string literals on separate lines to automatically concatenate. If the enclosing parens aren't there, you have to dedent the second line or you get a syntax error, and that just looks weird.